.gitmodules

```diff
@@ -2,8 +2,8 @@
 	path = build-infra
 	url = git@github.com:Metaswitch/clearwater-build-infra.git
 [submodule "modules/common"]
-	path = modules/common
+	path = common
 	url = git@github.com:Metaswitch/python-common.git
 [submodule "modules/telephus"]
-	path = modules/telephus
+	path = telephus
```

Comment:
What are the ramifications of this?  I assume people need to `git submodule sync` at least?